tests/artifacts/test_artifacts.py

```diff
@@ -466,3 +466,19 @@ def test_inline_body():
     )
     assert artifact.spec.get_body() == "123"
     assert artifact.metadata.key == "y"
+
+
+def test_import_artifact_using_relative_path():
```

Comment:
now thinking about it I think this should be also moved to `test_project.py` as it tests the project's `import_artifact`